craft_store/auth.py

```diff
@@ -170,7 +176,7 @@ def get_credentials(self) -> str:
             encoded_credentials_string = self._keyring.get_password(
                 self.application_name, self.host
             )
-        except Exception as unknown_error:
+        except errors.CraftStoreError as unknown_error:
```

Comment:
Please return this to `Exception` and put an appropriate linter silencing comment here. `keyring` won't raise store errors.